src/libopensc/types.h

```diff
@@ -225,10 +225,10 @@ typedef struct sc_acl_entry {
 #define SC_FILE_EF_CYCLIC		0x06
 #define SC_FILE_EF_CYCLIC_TLV		0x07
 
-/* Data compression */
-#define SC_FILE_COMPRESSED_AUTO		0x08
-#define SC_FILE_COMPRESSED_ZLIB		0x09
-#define SC_FILE_COMPRESSED_GZIP		0x0a
+/* File flags */
+#define SC_FILE_FLAG_COMPRESSED_AUTO		0x08
+#define SC_FILE_FLAG_COMPRESSED_ZLIB		0x09
+#define SC_FILE_FLAG_COMPRESSED_GZIP		0x0a
```

Comment:
```suggestion
#define SC_FILE_FLAG_COMPRESSED_AUTO		0x01
#define SC_FILE_FLAG_COMPRESSED_ZLIB		0x02
#define SC_FILE_FLAG_COMPRESSED_GZIP		0x04
```

Reply:
Thanks for review, fixed & rebased onto current master.